### ISSUE TITLE
Process buffer spreading from the current position

### ISSUE THREAD

**Issue by adithyaov** [abo-abo/avy#294]
Hi, I'm a new emacs user. I used to use easymotion a lot in Vim and am happy to see something very similar in emacs as well.

When processing the current view, Avy does it in a top to bottom fashion. Ie. smaller letter combinations always appear on the top of the view.
Consider the following,
* Cursor is at `s` denoted by `[s]` on line 2
```
This is line 1
This i[s] line 2
This is line 3
```
If we do an `avy-goto-word-0`, Something like this would appear,
* Note that this is made up. The letters and combinations may be different.
* Avy marks are enclosed within `{` and `}`
```
{a}his {b}s {c}ine {d}
{ea}his {eb}[s] {ec}ine {ed}
{fea}his {feb}s {fec}ine {fed}
```
Instead of this what if I want something like,
```
{ea}his {eb}s {ec}ine {ed}
{a}his {b}[s] {c}ine {d}
{fea}his {feb}s {fec}ine {fed}
```
Ie. The words closest to the current point should have the least amount of combinations of letters.
In other words, process the view starting from the current point, outwards.

Please let me know, if there is a way to emulate this, else, I'll be happy to make a PR with some assistance :-)

PS: I apologize if I did not use the terms correctly.

**Comment by abo-abo** [comment]
By default `avy-goto-char` is already configured in this way.

You can configure `avy-goto-word-0` like this:

```elisp
(setq avy-orders-alist
      '((avy-goto-char . avy-order-closest)
        (avy-goto-word-0 . avy-order-closest)))
```

**Comment by abo-abo** [comment]
I've put this info on the wiki: https://github.com/abo-abo/avy/wiki/defcustom

**Comment by adithyaov** [comment]
Aha! Thank you very much. I apologize, I seem to have missed it somehow.
